spec/features/visitor_can_create_an_account_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.feature "Visitor tries to create account" do
+  context "providing valid credentials" do
+    scenario  "account creation is successful" do
+      visit new_user_path
+
+      fill_in "Username", with: "NewUser"
```

Comment:
In your schema you have a column for 'name' and not 'username' for your 'users' table. This test will not be able to find the field 'Username' to fill in. Just change it to 'name' here in the test on line 8 and you will be good.